### PR TITLE
Home Page API creation

### PR DESCRIPTION
**Pull request by singh-atul**
This api accepts a post request at api/v1/all and return the list of categories available in the database

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,3 +182,45 @@ mysql> exit;
...
+```
+
+## Table Creation:
+
+Use the following command to create the tables :
+
+> Categories
+
+```
+CREATE TABLE `Categories` (
```
Comment (RohitMazumder) [inline README.md:203]: This is erronous - ERROR 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '' at line 7

Comment (RohitMazumder) [reply README.md:203]: Resolved in next commit - 7b6b666eb0b5fdc774c8debc28c4ee57f3bdda6c

File: README.md [hunk 2]
```diff
@@ -1,2 +1,226 @@
...
+  `ID` int NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) NOT NULL,
+  `CreatedAt` datetime DEFAULT CURRENT_TIMESTAMP,
+  `UpdatedAt` datetime DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY (`ID`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4;;
```
Comment (RohitMazumder) [inline README.md]: There are two semicolons at the end of this query, which creates an empty query. Please fix this.

Comment (singh-atul) [reply README.md]: Fixed

**Review by RohitMazumder — CHANGES_REQUESTED** [review]
Remove package-lock.json

**Review by RohitMazumder — APPROVED** [review]

**Pull request merged**